apps/wakunode2/external_config.nim

```diff
@@ -211,6 +211,11 @@ type
       defaultValue: ""
       name: "rln-relay-tree-path" }: string
 
+    rlnRelayBandwidthThreshold* {.
```

Comment:
From a user perspective, how could I set the proper value of this setting?

Reply:
Hmm, it's mentioned bytes/sec, so it should be according to their personal bandwidth constraints.

Reply:
im fine with having it as a cli flag, but imho this parameter should be fixed for the network and not "according to their personal bandwidth constraints". wouldn't this be dangerous?

if different nodes in the same network set different thresholds, then message validity (valid/invalid) wont be an objective truth, but something that will depend on each node (each node will have its own view on wether the message is valid or invalid). and for waku as a network, i thinnk this can cause troubles.

Reply:
I think this problem can be solved if we broadcast capabilities like waku_relay, however if you'd like to set it to a global value, it'll be a matter of prioritising resource restrained nodes vs an overall global stable network. wdyt?

Reply:
just to ensure we are on the same page:
* do we agree that these value has to be fixed for the whole network because of the above reasons?
* mind elaborating on "it'll be a matter of prioritising resource restrained nodes vs an overall global stable network"?

Reply:
1. yes, agree that the value has to be fixed for the whole network
2. follow up comment

Reply:
the network will be stable if we set a fixed bandwidth threshold for all nodes, however, some nodes which have a bandwidth lesser than that will blindly forward messages to other nodes. and if we implement peer scoring for rln, this opens another issue where we need to adjust peer score based on the other's bandwidth threshold